helpers.ts

```diff
@@ -143,6 +143,33 @@ export function createTable(headers: string[], data: string[][]): any {
 	return table;
 }
 
+export function createTableWithVersionsInformation(versionsInformation: IVersionInformation[]): any {
```

Comment:
Is this method used on more than one place? Can we add it to a command/service or somewhere else as it seems quite specific. In `helpers` we add only very common methods that  can be used in various cases.